src/chrome/content/rules/1and1-Internet.xml

```diff
@@ -1,77 +0,0 @@
-<!--
-	For other United Internet coverage, see United-Internet.xml.
-
-
-	Nonfunctional hosts in *1and1.com:
-
-		- devpartner ᵖ
-		- mirror ᵈ
-		- newsroom ʳ
-		- postmaster ᵈ
-		- press ʳ
-
-	ᵈ Dropped
-	ᵖ Plaintext reply
-	ʳ Refused
-
-
-	Problematic hosts in *1and1.com:
-
-		- about ᵐ
-		- blog ᵐ
-		- faq ᵐ
```

Comment:
This comment is also true, and yet you now rewrite it directly. Again, we generally do not like to break things.